stream_alert_cli/terraform_generate.py

```diff
@@ -235,38 +315,130 @@ def generate_cluster(**kwargs):
         'role_policy_prefix': cluster_name
     }
 
+
+def generate_cloudtrail(cluster_name, cluster_dict, config):
+    """Add the CloudTrail module to the Terraform cluster dict.
+
+    Args:
+        cluster_name [string]: The name of the currently generating cluster
+        cluster_dict [defaultdict]: The dict containing all Terraform config for
+                                    a given cluster.
+        config [dict]: The loaded config from the 'conf/' directory
+    """
+    modules = config['clusters'][cluster_name]['modules']
+    cloudtrail_enabled = bool(modules['cloudtrail']['enabled'])
+    cluster_dict['module']['cloudtrail_{}'.format(cluster_name)] = {
+        'account_id': config['global']['account']['aws_account_id'],
+        'cluster': cluster_name,
+        'kinesis_arn': '${{module.kinesis_{}.arn}}'.format(cluster_name),
+        'prefix': config['global']['account']['prefix'],
+        'enable_logging': cloudtrail_enabled,
+        'source': 'modules/tf_stream_alert_cloudtrail',
+        's3_logging_bucket': '{}.streamalert.s3-logging'.format(
+            config['global']['account']['prefix'])}
+
+
+def generate_flow_logs(cluster_name, cluster_dict, config):
+    """Add the VPC Flow Logs module to the Terraform cluster dict.
+
+    Args:
+        cluster_name [string]: The name of the currently generating cluster
+        cluster_dict [defaultdict]: The dict containing all Terraform config for
+                                    a given cluster.
+        config [dict]: The loaded config from the 'conf/' directory
+    """
+    modules = config['clusters'][cluster_name]['modules']
+    if modules['flow_logs']['enabled']:
+        cluster_dict['module']['flow_logs_{}'.format(cluster_name)] = {
+            'source': 'modules/tf_stream_alert_flow_logs',
+            'destination_stream_arn': '${{module.kinesis_{}.arn}}'.format(cluster_name),
+            'flow_log_group_name': modules['flow_logs']['log_group_name']}
+        for input in ('vpcs', 'subnets', 'enis'):
```

Comment:
we should avoid using the python builtin `input` as a var name. I'm surprised it even lets you do this